pkg/storkctl/migrationschedule.go

```diff
@@ -162,17 +233,28 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 		},
 	}
 	createMigrationScheduleCommand.Flags().StringSliceVarP(&namespaceList, "namespaces", "", nil, "Specify the comma-separated list of namespaces to be included in the migration")
+	createMigrationScheduleCommand.Flags().StringToStringVar(&namespaceSelectors, "namespace-selectors", nil, "Resources in the namespaces with the specified namespace labels will be migrated. All the labels provided in this option will be OR'ed")
 	createMigrationScheduleCommand.Flags().StringVarP(&clusterPair, "cluster-pair", "c", "", "Specify the name of the ClusterPair in the same namespace to be used for the migration")
+	createMigrationScheduleCommand.Flags().StringVar(&adminClusterPair, "admin-cluster-pair", "", "Specify the name of the admin ClusterPair used to migrate cluster-scoped resources, if the ClusterPair is present in a non-admin namespace")
 	createMigrationScheduleCommand.Flags().BoolVarP(&excludeResources, "exclude-resources", "", false, "If present, Kubernetes resources will not be migrated")
-	createMigrationScheduleCommand.Flags().BoolVarP(&excludeVolumes, excludeVolumesFlag, "", false, "If present, the underlying Portworx volumes will not be migrated. This is the only allowed and default behaviour in sync-dr use cases or if storage options are not provided in the cluster pair.")
+	createMigrationScheduleCommand.Flags().BoolVarP(&excludeVolumes, excludeVolumesFlag, "", false, "If present, the underlying Portworx volumes will not be migrated. This is the only allowed and default behaviour in sync-dr use-cases or if storage options are not provided in the cluster pair")
```

Comment:
I agree that this description is a bit long and confusing.
The issue here is we are determining sync or async on basis of length of options map in the cluster-pair.
If len(options) ==0 then set default value of includeVolumes to false.

options are empty for sync-dr and have ip,token etc for async-dr when we create the clusterpair using storkctl.

However options can be empty in case the customer is using the older clusterpair generate command and forgets to provide options while creating a clusterpair for async-dr.

That's why I had modified the message.

Maybe we can say here that for async-dr default includeVolumes is true and for sync-dr includeVolumes is false.
And somewhere in the documentation we will have to add that in case clusterpair is not configured properly and storage options are not provided, includeVolumes will be set to false since Storage pairing will have failed in the clusterpair.

Any suggestions on what we should do in this case then ??

Reply:
Something like this should work.. For async-dr default includeVolumes value  is true and for sync-dr includeVolumes is false. For sync-dr includeVolumes = true is an invalid configuration because....
Wdyt?

Reply:
Have discussed with Priyanshu.
Will update the exclude-volumes flag's description to 
`If present, the underlying Portworx volumes will not be migrated. This is the only allowed and default behaviour in sync-dr use-cases or when storage options are not specified in the cluster pair`